ansible/group_vars/all/config.yml

```
@@ -166,6 +166,33 @@ configuration_all:
     server: true
     client: false
     nuxtclient: false
+  # Identity Management
+  FEATURE_IDENTITY_MANAGEMENT_ENABLED:
+    value: 'false'
+    server: true
+    client: false
+    nuxtclient: false
+  IDENTITY_MANAGEMENT__TENANT:
+    value: 'dBildungscloud'
```

Shoud be bound to the ansible variable NAMESPACE.
It's more individual for our instances
```suggestion
    value: "{NAMESPACE}"
```

---

done